client/blocks/comment-detail/comment-detail-reply.jsx

```diff
@@ -35,11 +35,23 @@ export class CommentDetailReply extends Component {
 		return Math.max( TEXTAREA_HEIGHT_FOCUSED, textareaHeight );
 	}
 
-	getTextareaPlaceholder = () => this.props.authorDisplayName
-		? this.props.translate( 'Reply to %(commentAuthor)s…', {
-			args: { commentAuthor: this.props.authorDisplayName }
-		} )
-		: 'Reply to comment…';
+	getTextareaPlaceholder = () => {
+		const { authorDisplayName, commentStatus, translate } = this.props;
+
+		if ( 'approved' !== commentStatus ) {
+			return authorDisplayName
+				? translate( 'Approve and reply to %(commentAuthor)s…', {
+					args: { commentAuthor: authorDisplayName }
+				} )
+				: translate( 'Approve and reply to comment…' );
+		}
+
+		return authorDisplayName
+			? translate( 'Reply to %(commentAuthor)s…', {
+				args: { commentAuthor: authorDisplayName }
+			} )
+			: translate( 'Reply to comment…' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **95** times:
`translate( 'Reply to Comment' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/comments/approve-and-reply)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).